packages/react-examples/src/office-ui-fabric-react/DetailsList/DetailsList.GroupedV2.Large.Example.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import {
+  DetailsHeader,
+  DetailsList,
+  IColumn,
+  IDetailsHeaderProps,
+  IGroup,
+} from 'office-ui-fabric-react/lib/DetailsList';
+import { GroupedListV2_unstable as GroupedListV2 } from 'office-ui-fabric-react/lib/GroupedListV2';
```

Comment:
_nit:_ You can pull these up into one import statement

```suggestion
import {
  DetailsHeader,
  DetailsList,
  IColumn,
  IDetailsHeaderProps,
  IGroup,
  GroupedListV2_unstable as GroupedListV2,
} from 'office-ui-fabric-react/lib/DetailsList';
```